testplan/testing/multitest/entries/base.py

```diff
@@ -272,13 +272,18 @@ class Graph(BaseEntry):
     """Create a graph for the report."""
     def __init__(self, graph_type, graph_data, description=None,
                  series_options=None, graph_options=None):
+        valid_graph_types = ['Line', 'Scatter', 'Bar', 'Whisker',
+                             'Contour', 'Hexbin']
+        valid_chart_types = ['Pie']
+
         self.graph_type = graph_type
         self.graph_data = graph_data
         self.series_options = series_options
         self.graph_options = graph_options
-        if graph_type != 'Pie':
-            self.type = 'Graph'
-        else:
-            self.type = 'DiscreteChart'
+        self.type = 'Graph'
+        if graph_type in valid_chart_types:
+            self.discrete_chart = True
+        elif graph_type in valid_graph_types:
+            self.discrete_chart = False
```

Comment:
What happens if the graph_type is not in `valid_chart_types` or `valid_graph_types`? I think you should probably raise a `ValueError` since we can only plot these types.